compat/src/index.js

```diff
@@ -116,10 +116,6 @@ function upgradeToVNodes(arr, offset) {
 		if (Array.isArray(obj)) {
 			upgradeToVNodes(obj);
 		}
-		else if (obj && typeof obj==='object' && !isValidElement(obj) && ((obj.props && obj.type) || obj.text!=null)) {
```

Comment:
This branch is not needed anymore because every `vnode` that passes through `h` in compat or core will have the `$$typeof`-property now .

Reply:
The entire `upgradeToVNodes` function is a no-op now that this branch has been removed, all it does is call itself recursively.

Reply:
Ohh thanks for the pointer. Not sure why I didn't see that :joy: